### ISSUE TITLE
Implement viewing tiers 

### ISSUE THREAD

**Issue by fabiancook** [socialbaking/karma#71]
Some information should be only be available to those who have provided verified information, or have provided consistently data. 

A user should be able to see information for a window after the period that they supplied information. 

If the user is not supplying information, they may not be able to purchase the products, meaning they shouldn't be able to view the information. 

Users with verified details should be able to see more information than a brand new user.
Users that have provided details about a product should be able to see details about a product for a period of time, and continue to see standard details for the product going forward. 
Users with verified details that have provided details about any product should be able to see more insights into products
Users that have consistently provided details about products should be able to see more and more metrics and insights. 

Users with verified details that have contributed to the community and to metrics should have complete access. 

----

TLDR Collect karma to see metrics
